index.html

```diff
@@ -1 +1,152 @@
+                    <h3>Maryland Institute College of Art </h3>
+                    <h4>2019–Present | Baltimore, MD</h4>
+                    <ul>
+                        <li>MFA candidate in Graphic Design (Expected May 2021)</li>
+                        <li>Merit Scholarship Recipient</li>
+                        <li>Graduate Teaching Intern for Photo Imaging and Design for User Experience</li>
+                    </ul>
+                    <h3>Kookmin University </h3>
+                    <h4>2010–2014 | Seoul, Korea BFA in Painting & Graphic Design</h4>
+                    <p>Honors Scholarship (Spring 2011, Fall 2012)</p>
+                    <h3>Coventry University </h3>
+                    <h4>Fall 2011 | Coventry, UK</h4>
+                    <p>Exchange Program in Fine Arts </p>
+                </div>
+                <div class="inner-col" id="award">
+                    <h2>Awards</h2>
+                    <h3><a href="https://www.ux-design-awards.com/projekte/10bytes">User Experience Design Awards |
+                            New Talent</a></h3>
+                    <h4>2020 | Berlin, Germany
+                    </h4>
+                    <p>
+                        Winner, 10Bytes </p>
+                    <h3>
+                        <a href="http://contests.gdusa.com/gdusa-contest-winner?cc=aida20&ids=0|10780&iy=&im=">
+                            Graphic Design USA </a>
+                    </h3>
+                    <h4>2020 | New York, NY
+                    </h4>
+                    <p>
+                        Winner, CHIVE </p>
+                    <h3><a href="https://www.graphis.com/entry/e1f3bb10-1f2d-4e68-8feb-56f2dad955b0/">Graphis Poster
+                            Annual 2021</a></h3>
+                    <h4>2020 | New York,
+                        NY</h4>
+                    <p>
+                        Gold, More Than a Vessel </p>
+                    <h3><a href="https://100.sta-chicago.org/winners/2019/typeface-flow">The Society of Typographic
+                            Art 100</a></h3>
+                    <h4>2019 | Chicago,
+                        IL</h4>
+                    <p>Winner,
+                        Typeface: Flow</p>
+                </div>
+                <div class="inner-col" id="skill">
+                    <h2>Skills</h2>
+                    <ul>
+                        <li>Product Design</li>
+                        <li>UX Research</li>
+                        <li>Rapid Prototyping</li>
+                        <li>Systems Thinking</li>
+                        <li>Affinity Diagramming</li>
+                        <li>Graphic Design</li>
+                        <li>Drawing/Visualizing</li>
+                        <li>Branding</li>
+                        <li>Photography</li>
+                        <li>Storybording</li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </div>
+</body>
```

Comment:
Just want to reiterate here that you've written some great, clean HTML. Bravo 👏🏻👏🏻👏🏻